src/CancelController.ts

```diff
@@ -1,23 +1,22 @@
-import {CancelTokenSource} from 'axios'
-const {CancelToken} = require('axios')
+export interface CancelControllerInterface {
```

Comment:
Why do you add Interface here? Why do you export it? We don't have plan to provide tools for create another CancelController in user land

Reply:
But you want to pass CancelController as param and you should provide typing for this.

Reply:
And you will use this CancelController in your code and you can see available methods

Reply:
I use CancelController as type